src/aura/BGE_ConfigurationWizard/BGE_ConfigurationWizard.design

```diff
@@ -0,0 +1,4 @@
+<design:component>
+    <design:attribute name="isReadOnly" default="true" label="Is Read Only"
```

Comment:
note: This is only used for templates - if we can get rid of templates we don't need this